### PR TITLE
Added first two sections

### PR DESCRIPTION
**Pull request by ArkanKovanAbdullah**
## First section
- Added first section.
- First section: navigation in flexbox.
- logo.
- hamburger menu.

## Second section
- Added section in flex box. 
- Made flex box direction in column.
- Added three different texts. 
- Added icons.

## Icons

-Icons were added using font awesome.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles.css">
+        <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+        <script src="https://kit.fontawesome.com/6776bca25f.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        
+        <header>
+            <nav class="beflex">
+                <div class="navitems Logo" id="my-name">IDAE</div>
```
Comment (andgarzonmal) [inline index.html:20]: - [ ] Kyndly wrap the logo inside an `a` tag remember this is the link that will take the user to the main page.

- This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <i class="fa-solid fa-bars navitems fa-2xl"></i>
+            </nav>
+
+            <section id="secondsection"> 
+                <h2 id="firttext">I’m Arkan K. Abdullah <br> Glad to see <br> you!</h2>
```
Comment (andgarzonmal) [inline index.html:25]: - [ ]  Kyndly use an `h1` tag istead of an `h2` tag ♻️

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                <p id="ntext">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p id="connectbtn">LETS CONNECT</p>
+                <section id="nestedsection">
+                    <i class="fa-brands fa-twitter fa-lg nestedchild"></i>
+                    <i class="fa-brands fa-linkedin-in fa-lg nestedchild"></i>
+                    <i class="fa-solid fa-m fa-lg nestedchild"></i>
+                    <i class="fa-brands fa-github fa-lg nestedchild"></i>
+                    <i class="fa-solid fa-hand-peace fa-lg nestedchild"></i>
+                </section>
```
Comment (andgarzonmal) [inline index.html:34]: - [ ] Kyndly wrap the lit of icons into a `ul` tag and use a `li` tag per icon, also use an `a` tag to wrap the icons, apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Here is an example 

image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @ArkanKovanAbdullah✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by GiuseppeTG — APPROVED** [review]
Hi @ArkanKovanAbdullah  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights

✅ All changes implemented! 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**